### DISCUSSION TITLE
QRCode not working with .NETCore 6.0

### DISCUSSION THREAD

**Discussion by akosilataunglahat** [Shane32/QRCoder#410] [Q&A] [answered]
Hello @codebude  - would like to seek for your help as I'm using your library but having problem when I update my solution to .NETCore 6.0.. QRCode is not working anymore and upon checking it's not compatible with .NETCore 6.0. I need this as I'm using it with a Logo. I can't add a logo anymore with the other method on your library as there's no property on it only those methods which are not compatible with .NETCore 6.0? Any advise on this? Thank you in advance for your quick response.

**Comment by akosilataunglahat** [comment, marked as answer]
Please disregard my question. I got this working by downgrading the QRCoder version to 1.4.1.. Thank you. 👍

**Comment by Mksm-V** [comment]
Hello, @codebude! And yet, is there a solution for .NETCore 6.0?
